packages/storybook/stories/components/accordion/accordion.stories.tsx

```diff
@@ -0,0 +1,265 @@
+import { Text } from '@ovhcloud/ods-react';
+import { type Meta, type StoryObj } from '@storybook/react';
+import React, { useState } from 'react';
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  type AccordionProp,
+  AccordionTrigger,
+} from '../../../../ods-react/src/components/accordion/src';
+import { CONTROL_CATEGORY } from '../../../src/constants/controls';
+import { excludeFromDemoControls, orderControls } from '../../../src/helpers/controls';
+
+type Story = StoryObj<AccordionProp>;
+type DemoArg = Partial<AccordionProp>
```

Comment:
not needed if you don't use a mix of prop from other component that the main one